### ISSUE TITLE
After resharing a post one can still find the reshare link in reshares

### ISSUE THREAD

**Issue by svbergerem** [diaspora/diaspora#6594]
Works in all streams and the single post view. We should check if the user reshared the original post instead of checking it for the reshare.
##

**Comment by goobertron** [comment]
I've just come across something related to this. My case:

- I reshared a reshare post (https://pod.orkz.net/posts/2754041) from the stream. The 'Reshare' link disappeared from under the post, as expected.
- Then I went to the SPV for that reshare, and saw the (unhighlighted) reshare icon, which should have been either disabled/hidden or highlighted as 'already reshared'.
- Went to the original post (https://pod.orkz.net/posts/2753332) and saw the reshare icon highlighted, and my avatar in the list of resharers (as expected).

If it's possible to remove the 'Reshare' link from *all* reshares of the original post that appear in the stream when removing it from the reshare you've reshared, that would be good.

I think it would also be good, when viewing a reshare in the SPV, to place the reshare icon against the original post rather than against the reshare, to make it clear that you'll be resharing the original post rather than the reshare.
